### DISCUSSION TITLE
[Question] Form Action Submission

### DISCUSSION THREAD

**Discussion by devappbox** [sveltejs/kit#8645] [Q&A] [answered]
### Describe the problem

Hello, i am new to SvelteKit. I am using  the use:enhance form action. My question is how to display a dialog box so that i can choose to continue or cancel a submission

``````
<form
  method="POST"
  use:enhance={({ form, data, action, cancel }) => {

    //Call or Show Dialog Box with options to continue or cancel submission   
   
    return async ({ result, update }) => {
    };
  }}
>
``````

Thank you for the any advice

**Comment by Conduitry** [comment, marked as answer]
Please don't open issues for support requests, use the discussions feature instead - transferring

**Comment by teemingc** [comment]
Here's an example I whipped up. Not sure if this is right but I hope it helps give an idea of what it could look like.

https://stackblitz.com/edit/sveltejs-kit-template-default-gfns2v?file=src/routes/+page.svelte

**Reply by devappbox** [reply]
> Here's an example I whipped up. Not sure if this is right but I hope it helps give an idea of what it could look like.
> 
> https://stackblitz.com/edit/sveltejs-kit-template-default-gfns2v?file=src/routes/+page.svelte

Thank you for your insight, i will try it.
